src/constants.ts

```diff
@@ -15,5 +15,7 @@ export const AGARTEX_SERVICE_COMPILATION_URL = AGARTEX_SERVICE_BASE_URL + '/comp
 export const AGARTEX_SERVICE_SESSIONS_URL = AGARTEX_SERVICE_BASE_URL + '/sessions';
 export const AGARTEX_SERVICE_USERS_URL = AGARTEX_SERVICE_BASE_URL + '/users';
 
+export const COLLABORATION_SOCKET_URL = 'ws://localhost:6789/';
```

Comment:
Change it to something like `AGARTEX_SERVICE_BASE_URL+'/websocket''` (same port as other endpoints)

Reply:
going for `export const AGARTEX_COLLABORATION_URL = AGARTEX_SERVICE_BASE_URL + '/collaboration'`

Reply:
Remember to keep `ws://` prefix

Reply:
uhh right, wait

Reply:
I reorganized env variables, check out 'constants.ts' and `envExample'